src/languages/gml.js

```diff
@@ -2599,8 +2774,7 @@ export default function(hljs) {
     "vk_subtract",
     "vk_tab",
     "vk_up"
-  ];
-  const LANGUAGE_VARIABLES = [
+  const LANGUAGE_VARIABLES = [];
     "alarm",
     "application_surface",
     "argument",
```

Comment:
Not sure how tests ever passed? This doesn't even look syntactically correct?  Did it get warped somehow?

Reply:
It was auto-generated, seems that at some point I overlooked that it started to generate invalid JS code. Fixed now.